src/packagedcode/go_mod.py

```diff
@@ -31,10 +30,48 @@
 import re
 
 import attr
+from packageurl import PackageURL
+
 
+TRACE = False
 
+logger = logging.getLogger(__name__)
+
+if TRACE:
+    import sys
+    logging.basicConfig(stream=sys.stdout)
+    logger.setLevel(logging.DEBUG)
+
+
+@attr.s()
+class GoModules(object):
```

Comment:
Why plural name?

Reply:
I was reading https://blog.golang.org/using-go-modules documentation. `Go Modules` was written in that documentation. So I used that name :)

Reply:
OK, but your instance still represents a single module object, not many ones?

Reply:
I represents the Single module. I have fixed this.